### PR TITLE
add: `wezterm-app`

### PR DESCRIPTION
**Pull request by smokeythemonkey**
## Progress 

- [x] Edit packagelist
- [x] Add initial pacscript
- [ ] Contact devs
- [x] Add maintainer to pacscript

### CODE REVIEW

**Comment by Henryws** [comment]
Uh, fix `packagelist`

**Comment by smokeythemonkey** [comment]
Whoops :D. Done :+1:

**Comment by Henryws** [comment]
Fix the `packagelist` so the *only* thing being added is `wezterm-app`, instead of moving around other stuff.

File: packages/wezterm-app/wezterm-app.pacscript [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name="wezterm-app"
+pkgname="wezterm"
+version="20221119-145034-49b9839f"
+description="WezTerm is a GPU-accelerated cross-platform terminal emulator and multiplexer written by @wez and implemented in Rust"
+url="https://github.com/wez/wezterm/releases/download/${version}/WezTerm-${version}-Ubuntu18.04.AppImage"
+hash=""
```
Comment (Henryws) [inline packages/wezterm-app/wezterm-app.pacscript:6]: Why is this empty?

File: packages/wezterm-app/wezterm-app.pacscript [hunk 2]
```diff
@@ -0,0 +1,37 @@
+name="wezterm-app"
+pkgname="wezterm"
+version="20221119-145034-49b9839f"
+description="WezTerm is a GPU-accelerated cross-platform terminal emulator and multiplexer written by @wez and implemented in Rust"
```
Comment (Henryws) [inline packages/wezterm-app/wezterm-app.pacscript:4]: Don't use the name of the package in the description.

File: packages/wezterm-app/wezterm-app.pacscript [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+repology=("project: wezterm")
+
+prepare() {
+  true
+}
+
+build() {
+  true
+}
+
```
Comment (Henryws) [inline packages/wezterm-app/wezterm-app.pacscript:18]:
```suggestion
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/wezterm-app/wezterm-app.pacscript (hunk 1 above)
Comment (smokeythemonkey) [reply packages/wezterm-app/wezterm-app.pacscript]: honestly just forgot

File: packages/wezterm-app/wezterm-app.pacscript [hunk 4]
```diff
@@ -0,0 +1,28 @@
+name="wezterm-app"
+pkgname="wezterm"
```
Comment (Henryws) [inline packages/wezterm-app/wezterm-app.pacscript:2]:
```suggestion
gives="wezterm"
```

**Review by wizard-28 — APPROVED** [review]

File: packages/wezterm-app/wezterm-app.pacscript [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+install() {
+  # Install AppImage
+  sudo install -Dm755 "WezTerm-${version}-Ubuntu18.04.AppImage" "${pkgdir}/usr/bin/${pkgname}"
+
+  # Download icon
+  wget -q https://github.com/wez/wezterm/blob/main/assets/icon/terminal.png
+  sudo install -Dm644 "terminal.png" "${pkgdir}/usr/share/icons/hicolor/128x128/apps/org.wezfurlong.wezterm.png"
+
+  # Install .desktop file
+  #wget -q "https://raw.githubusercontent.com/pacstall/pacstall-programs/master/packages/${name}/${pkgname}.desktop"
```
Comment (Henryws) [inline packages/wezterm-app/wezterm-app.pacscript:21]: Uncomment these

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — APPROVED** [review]

**Pull request merged**